app.js

```diff
@@ -125,37 +125,22 @@ app.get('/data/bootstrapcdn.json', function (req, res) {
 
 // Merge in variable options for sitemap.
 function sitemapOptions (options) {
-    options.http = 'https';
-
     if (env === 'production') {
```

Comment:
Isn't this whole check redundant now? `options` is returned at the end as far as I can tell.